keyboards/handwired/orbweaver/keymaps/default/keymap.c

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2023 a_marmot
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see &lt;http://www.gnu.org/licenses/&gt;.
+*/
+
+#include QMK_KEYBOARD_H
+#include <is31fl3731.h>
```

Comment:
This shouldn't be needed.
```suggestion
```

Reply:
Isn't this required for the use of the IS31 matrix controller?

Reply:
No, it's included internally.

Reply:
Do I need an #include quantum.h somewhere else?